hazelcast-spring/src/main/resources/hazelcast-spring-3.7.xsd

```diff
@@ -2420,7 +2420,7 @@
         <xs:attribute name="enabled" default="true" type="parameterized-boolean"/>
     </xs:complexType>
     <xs:complexType name="memory-size">
-        <xs:attribute name="value" type="xs:int" default="128"/>
+        <xs:attribute name="value" type="intMin0" default="128"/>
```

Comment:
I think, `intMin1` (positive integer) might be better instead of `intMin0` (non negative integer)